pages/index.vue

```diff
@@ -106,7 +106,7 @@
         </v-row>
 
         <!-- Download the App -->
-        <v-row id="download-app" class="mt-12">
+        <v-row id="download-app" class="px-12 mt-12">
```

Comment:
I think you will probably want to change this to `px-4 px-sm-12` (I think that's what's in my other branch on the other sections)